Flaky integration test on 32bit architecture

I only just noticed this, apparently we have a flaky test that triggers particularly often on the 32bit travis build. It's the `test_reconnect_normal` test, which fails to grab `CHANNELD_NORMAL` from the daemon funding the channel:

```
____________________ LightningDTests.test_reconnect_normal _____________________
self = <test_lightningd.LightningDTests testMethod=test_reconnect_normal>
    @unittest.skipIf(not DEVELOPER, "needs DEVELOPER=1")
    def test_reconnect_normal(self):
        # Should reconnect fine even if locked message gets lost.
        disconnects = ['-WIRE_FUNDING_LOCKED',
                       '@WIRE_FUNDING_LOCKED',
                       '+WIRE_FUNDING_LOCKED']
        l1 = self.node_factory.get_node(disconnect=disconnects)
        l2 = self.node_factory.get_node()
        ret = l1.rpc.connect(l2.info['id'], 'localhost', l2.info['port'])
    
>       self.fund_channel(l1, l2, 10**6)
tests/test_lightningd.py:1889: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_lightningd.py:254: in fund_channel
    l1.daemon.wait_for_log('-> CHANNELD_NORMAL')
tests/utils.py:163: in wait_for_log
    return self.wait_for_logs([regex], timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <utils.LightningD object at 0xf6abf88c>, regexs = ['-> CHANNELD_NORMAL']
timeout = 60
    def wait_for_logs(self, regexs, timeout=60):
        """Look for `regexs` in the logs.
    
            We tail the stdout of the process and look for each regex in `regexs`,
            starting from last of the previous waited-for log entries (if any).  We
            fail if the timeout is exceeded or if the underlying process
            exits before all the `regexs` were found.
    
            """
        logging.debug("Waiting for {} in the logs".format(regexs))
        exs = [re.compile(r) for r in regexs]
        start_time = time.time()
        pos = self.logsearch_start
        initial_pos = len(self.logs)
        while True:
            if time.time() > start_time + timeout:
                print("Can't find {} in logs".format(exs))
                with self.logs_cond:
                    for i in range(initial_pos, len(self.logs)):
                        print("  " + self.logs[i])
                for r in exs:
                    if self.is_in_log(r):
                        print("({} was previously in logs!)".format(r))
>               raise TimeoutError('Unable to find "{}" in logs.'.format(exs))
E               TimeoutError: Unable to find "[re.compile('-> CHANNELD_NORMAL')]" in logs.
tests/utils.py:139: TimeoutError
```
It appears that we have an incomplete transition to `CHANNELD_NORMAL` for the funding node.

Full logs [here](https://gist.github.com/cdecker/0fafccce026aebc9f719da8049d2813e)

--- a/tests/test_lightningd.py
+++ b/tests/test_lightningd.py
@@ -2544,7 +2544,6 @@ class LightningDTests(BaseLightningDTests):
         # Just to be sure, second openingd hand over to channeld.
         l2.daemon.wait_for_log('lightning_openingd.*REPLY WIRE_OPENING_FUNDEE_REPLY with 2 fds')
 
-    @unittest.skip("temporarily disabled due to flaky behavior, issue #468")
     @unittest.skipIf(not DEVELOPER, "needs DEVELOPER=1")
     def test_reconnect_normal(self):
         # Should reconnect fine even if locked message gets lost.

